packages/babel-helper-member-expression-to-functions/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@babel/helper-member-expression-to-functions",
```

Comment:
Author is missing. I guess you can put your name into it?

Reply:
Done.